mollie_subscription_ept/controllers/main.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+import werkzeug
+from odoo import fields, http
+from odoo.http import request, Response
+from odoo.tools.json import scriptsafe as json_scriptsafe
+
+_logger = logging.getLogger(__name__)
+
+
+class WebsiteSale(http.Controller):
+
+    @http.route(['/clear_warn'], type='json', auth="public", website=True)
+    def clear_warning(self, **post):
+        hide_warn = post.get('hide_warn')
+        if hide_warn:
```

Comment:
The function body can be replaced by:
request.session['show_wishlist_warn'] = request.session['show_in_cart_warn'] = not(post.get('hide_warn'))
return True